src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java

```diff
@@ -562,11 +571,12 @@ private void bootstrapVshard() {
     @Override
     protected void containerIsStarted(InspectContainerResponse containerInfo, boolean reused) {
         super.containerIsStarted(containerInfo, reused);
+        int secondsToWait = 120;
```

Comment:
This is constant and you can move it to class level. And it's better to rename it to something like that `TWO_MINUTES`